hw3/src/tree.cpp

```diff
@@ -64,7 +64,7 @@ void order(string a[], int n)
     // Don't use return on the recursion here bc we gotta go on each time? 
     // I know this is gonna mean it gets lost but also this is a void function?
     order(a, firstNotLess);
-    order(a + firstNotLess, n - firstGreater);
```

Comment:
Otherwise, you will lose some value at the end, because from firstNotLess to the end is `n - firstNotLess`.